src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -1569,6 +1569,11 @@ internal TypeAttributes Flags
             }
         }
 
+        public override bool AreLocalsZeroed
```

Comment:
>public override bool AreLocalsZeroed [](start = 8, length = 36)

sealed?

Reply:
Why use `sealed` here but not on the other `override` in this type?

Reply:
> Why use sealed here but not on the other override in this type? 

The answer might depend on each specific override. Also this is might be just an oversight, the code is pretty old, we were still learning what things are important 

---
In reply to: [172899607](https://github.com/dotnet/roslyn/pull/25276#discussion_r172899607) [](ancestors = 172899607)